feature-libs/pickup-in-store/core/config/pickup-in-store-config.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { Config } from '@spartacus/core';
+
+@Injectable({
+  providedIn: 'root',
+  useExisting: Config,
+})
+export abstract class PickupInStoreConfig {
+  pickupInStore?: {
+    consentTemplateId?: string;
```

Comment:
where this config is used? why we need it?

Reply:
Hi,Its used in 
/Users/I554698/Desktop/HOME/gitRepos/spartacus/feature-libs/pickup-in-store/core/pickup-in-store-core.module.ts
Its used for consent.